packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -242,9 +242,45 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     );
   }
 
-  // Render Items
+  // Render items
   @autobind
   private _onRenderItem(item: IDropdownOption): JSX.Element {
+    if (item.text === '-') {
```

Comment:
I see we're [doing the same in ContextualMenu](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx#L245) but this does seem odd to me. I'd rather see people explicitly specify `itemType: DropdownMenuItemType.Divider` as they have to do with headers than having this shortcut that could create an unintentional divider.